Library/Formula/siege.rb

```diff
@@ -2,8 +2,8 @@
 
 class Siege < Formula
   homepage 'http://www.joedog.org/index/siege-home'
-  url 'http://download.joedog.org/siege/siege-3.0.7.tar.gz'
-  sha256 'c651e2ae871cc680eb375f128b4809e97ceecf367f6bd65c3df00603fbceed4e'
+  url 'http://download.joedog.org/siege/siege-3.0.9.tar.gz'
+  sha256 '82376eb466414ef4872a979a372972658df9813778ee8572341d4736ed30cb8f'
   revision 1
```

Comment:
The `revision 1` needs removed on version updates, sorry for the inconvenience! Thanks!